library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,34 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+It is clearly undesirable for a dependency tree to contain conflicts.
+Unfortunately, as soon as any library in a dependency tree introduces a breaking
+change, consumers can't use the latest versions of those libraries together. To
+support users who try to generally use the latest versions of their
```

Comment:
This sentence can be cut down a bit. Maybe

To enable clients to depend on the latest versions of their dependencies,  breaking changes should be propagated as quickly as possible.

Reply:
Sounds good.